tools/GenerateAzureSiteExtension/SyncAzureAppServicesPackage.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using System.IO;
+using Datadog.Core.Tools;
+using PrepareRelease.Tools;
+
+namespace GenerateAzureSiteExtension
+{
+    public static class SyncAzureAppServicesPackage
+    {
+        private static string _tracerHomeDirectory = Path.Combine(EnvironmentTools.GetSolutionDirectory(), "deploy", "AzureAppServices");
+
+        public static void Run()
+        {
+            var pathVariableKey = "DD_AAS_PACKAGING_PATH";
+            var packagingPath = Environment.GetEnvironmentVariable(pathVariableKey);
+            if (string.IsNullOrWhiteSpace(packagingPath))
+            {
+                Console.WriteLine($"WARNING: You are using the default output path of: {_tracerHomeDirectory}");
+                Console.WriteLine($"WARNING: If you wish to sent to a specific path, set the {pathVariableKey} environment variable.");
+            }
+            else
+            {
+                _tracerHomeDirectory = packagingPath;
+            }
+
+            CopyTargetFrameworkPrivateBin("net45");
+            CopyTargetFrameworkPrivateBin("net461");
+            CopyTargetFrameworkPrivateBin("netstandard2.0");
+            CopyNativeProfiler("x86");
+            CopyNativeProfiler("x64");
+            CopyIntegrationsJson();
+        }
+
+        private static void CopyTargetFrameworkPrivateBin(string frameworkMoniker)
+        {
+            var outputDirectory = Path.Combine(_tracerHomeDirectory, frameworkMoniker);
+
+            if (Directory.Exists(outputDirectory))
+            {
+                Directory.Delete(outputDirectory, recursive: true);
+            }
+
+            Directory.CreateDirectory(outputDirectory);
+
+            var sourceFiles = DependencyHelpers.GetTracerReleaseBinContent(frameworkMoniker);
+
+            foreach (var sourceFile in sourceFiles)
+            {
+                var fileName = Path.GetFileName(sourceFile);
+                var destinationPath = Path.Combine(outputDirectory, fileName);
+                File.Copy(sourceFile, destinationPath);
+            }
+        }
+
+        private static void CopyNativeProfiler(string architecture)
+        {
+            var outputDirectory = Path.Combine(_tracerHomeDirectory, architecture);
+
+            if (Directory.Exists(outputDirectory))
+            {
+                Directory.Delete(outputDirectory, recursive: true);
+            }
+
+            Directory.CreateDirectory(outputDirectory);
+
+            var sourceFiles = DependencyHelpers.GetProfilerReleaseBinContent(architecture);
```

Comment:
I think we need the output from `publish` which includes all the dependencies in netcore.

Related: I wrote a [batch script](https://github.com/DataDog/dd-trace-dotnet/blob/lpimentel/aspnetcore-diagnosticsource-no-di/build-profiler-home.bat) that does something similar. It builds these projects and has the compiler publish the output to a single folder to be used with `DD_DOTNET_TRACER_HOME`. I needed this when testing the CoreMVC rewrite.

Reply:
Yeah, I think a dotnet publish and getting the subdirectory should do it.

Reply:
Can we figure out the pipeline stuff in another PR? I'd like to close this one out.

Reply:
@lucaspimentel The batch script link is broken by the way

Reply:
Oops, my bad. I removed it from my PR.